keps/sig-node/3063-dynamic-resource-allocation/README.md

```diff
@@ -1989,21 +1989,32 @@ new "ResourcePlugin" type will be used in the Type field of the
 [PluginInfo](https://pkg.go.dev/k8s.io/kubelet/pkg/apis/pluginregistration/v1#PluginInfo)
 response to distinguish the plugin from device and CSI plugins.
 
-Under the advertised Unix Domain socket the kubelet plugin provides the following
-gRPC interface. It was inspired by
+Under the advertised Unix Domain socket the kubelet plugin provides one of the
+following supported gRPC interface versions. It was inspired by
 [CSI](https://github.com/container-storage-interface/spec/blob/master/spec.md),
 with “volume” replaced by “resource” and volume specific parts removed.
 
+Key difference between interface versions:
+
+- [v1alpha2](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kubelet/pkg/apis/dra/v1alpha2/api.proto)
+interface provides resource claim information to kubelet plugin on one at a
+time basis. **NB! v1alpha2 will be deprecared, switch to v1alpha3**
```

Comment:
What is this `NB!`?  It just renders as the letters NB here:
https://github.com/kubernetes/enhancements/pull/4164/files#diff-8fa237d276346416c2aafa209f721e43c9762f59cabec234eafafe01694de3ee

Reply:
An acronym signifying the importans, similar to `NOTE!` or `MIND THAT...`
https://en.wikipedia.org/wiki/Nota_bene

Reply:
Let's remove it. I cant be alone in never seeing this acronym before and it doesn't seem to render as anything special in the markdown.

Reply:
Done.